halo/hooks/object.cpp

```diff
@@ -5,9 +5,13 @@
  */
 
 #include "../../hooker/hooker.hpp"
-#include "../memory/object.hpp"
+#include "../../hooker/function_pointer_safe.hpp"
+#include "../memory/gamestate/object/object.hpp"
+#include "../memory/behavior_definition.hpp"
+#include <windows.h>
+#include <cstring>
 
-Signture(true, sig_object_update,
+Signature(true, sig_object_update,
```

Comment:
Why are these values significant? A comment explaining these values would be very helpful here.

Reply:
I'll instead rename them to be more clear in the next pull request where they are actually used. Then I'll document what a signature is and what it's for in a readme down the line.

Reply:
I'll also update the comments there so they actually mean something.